### PR TITLE
Minesweeper

### PR DESCRIPTION
**Pull request by christopolise**
Made classic windows *barf* game.

### CODE REVIEW

File: demos/minesweeper/main.py [hunk 1]
```diff
@@ -0,0 +1,639 @@
...
+                # elif (i == 0) and ((j >= 1) and (j < (scale - 1))):
+                #     print("l" + " ", end="")
+
+                # # Right Border
+                # elif (i == (scale - 1)) and (j >= 1) and (j < (scale - 1)):
+                #     print("r" + " ", end="")
+
+                # # Center
+                # elif (i >= 1) and (i < (scale - 1)) and (j >= 1) and (j < (scale - 1)):
+                #     print(" " + " ", end="")
```
Comment (philipbl) [inline demos/minesweeper/main.py]: Can you remove these comments?

File: demos/minesweeper/main.py [hunk 2]
```diff
@@ -0,0 +1,639 @@
...
+
+    def check_win(self):
+        """Checks if the player has won the game"""
+        # Make sure there are 10 flags placed on the correct spots and that all the tiles have been discovered
+        # print(f"Num of placed:\t{len(self.placed_flags)}")
+        # print(
+        #     f"Placed:\t\t{self.placed_flags}\tMatched:\t{Counter(self.placed_flags) == Counter(self.mines)}"
+        # )
+        # print(f"Existing:\t{self.mines}")
+        # print(f"Discovered:\t{self.discovered_tiles}")
```
Comment (philipbl) [inline demos/minesweeper/main.py]: And these too?

**Review by philipbl — APPROVED** [review]
I left a few notes about removing comments. Great job!

**Review by apal6981 — APPROVED** [review]
Seems okay

**Pull request merged**